src/deepsparse/pipeline.py

```diff
@@ -176,29 +177,33 @@ def __init__(
 
         self._batch_size = self._batch_size or 1
 
-    def __call__(self, *args, **kwargs) -> BaseModel:
+    def __call__(self, *args, monitoring: bool = False, **kwargs) -> BaseModel:
```

Comment:
Should monitoring be a pipeline constructor arg? Will there be pipelines that sometimes monitor their forward calls and sometimes not?

Reply:
I'm okay with keeping this at the call level for now, nothing inherently changes about the pipeline object based on this variable and allows us to not have to reinitialize if for some reason we want to run with and without for the same pipeline

Reply:
Yeah but it makes the caller much easier if its an attribute. Probably don't even need run_with_monitoring in that case. And then in the server we can just pass that into constructor instead of having to specify monitoring=True or call run_with_monitoring.